manifests/facts.pp

```diff
@@ -41,10 +41,10 @@
   if $facts["os"]["family"] == "windows" {
     # on windows task scheduler prevent dupes already so no need to handle the PID here
     scheduled_task{"mcollective_facts_yaml_refresh":
-      ensure               => $cron_ensure,
-      command              => $rubypath,
-      arguments            => "'${scriptpath}' -o '${factspath}'",
-      trigger              => {
+      ensure    => $cron_ensure,
+      command   => $rubypath,
+      arguments => "'${scriptpath}' -o '${factspath}' --sleep '${fqdn_rand($refresh_interval, 'facts cronjob')}'",
```

Comment:
in my experience, changing schedule jobs is a shit show.  This is changing the schedule job :)

Are you convinced we need to do this as a parameter?  I guess it's the only way to really know we're spreading it over the entire spectrum.